driver/others/dynamic_arm64.c

```diff
@@ -125,8 +125,13 @@ extern gotoblas_t  gotoblas_THUNDERX2T99;
 extern gotoblas_t  gotoblas_TSV110;
 extern gotoblas_t  gotoblas_EMAG8180;
 extern gotoblas_t  gotoblas_NEOVERSEN1;
+#ifndef NO_SVE
 extern gotoblas_t  gotoblas_NEOVERSEV1;
 extern gotoblas_t  gotoblas_NEOVERSEN2;
+#else
+#define gotoblas_NEOVERSEV1 gotoblas_ARMV8;
+#define gotoblas_NEOVERSEN2 gotoblas_ARMV8;
```

Comment:
You've got some rogue semi-colons here:

```suggestion
#define gotoblas_NEOVERSEV1 gotoblas_ARMV8
#define gotoblas_NEOVERSEN2 gotoblas_ARMV8
```